pkg/apiserver/registry/networkpolicy/groupassociation/rest.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2020 Antrea Authors
```

Comment:
are we placing these under `pkg/apiserver/registry/networkpolicy` instead of `pkg/apiserver/registry/controlplane` on purpose? @tnqn

Reply:
No, there was a renaming, guess I or someone missed changing the package name.